apps/editor/src/app/panel/peerEditPanel.tsx

```diff
@@ -282,7 +282,10 @@ function PeerEditPanel({id, hostURL, onClose, onSave}: PeerEditPanelProps) {
                   {profile?.callToActionURL}
                 </DescriptionListItem>
                 <DescriptionListItem label={t('peerList.panels.callToActionImage')}>
-                  <img src={profile?.callToActionImage?.thumbURL || undefined} />
+                  <img
+                    src={profile?.callToActionImage?.thumbURL || undefined}
+                    alt="callToActionImage"
```

Comment:
```suggestion
                    alt="call to action image"
```

Ideally would be translated

Reply:
I changed it to translation